src/selectize.js

```diff
@@ -984,7 +984,7 @@ $.extend(Selectize.prototype, {
 		var self = this;
 
 		self.setTextboxValue('');
-		self.$control_input.css({opacity: 0, position: 'absolute', left: self.rtl ? 10000 : -10000});
+		self.$control_input.css({opacity: 0, position: 'absolute', left: self.rtl ? 10000 : 0});
```

Comment:
So if this is handling the position for RTL display, we probably want to change how this is done rather than pixel positioning. That can be a future taks.